src/mantle/database/factory/class-post-factory.php

```diff
@@ -75,6 +79,36 @@ public function with_thumbnail(): static {
 		);
 	}
 
+	/**
+	 * Attach a thumbnail to the post with an underlying file attachment.
+	 *
+	 * @param string $file   The file name to create attachment object from.
+	 * @param int    $width  The width of the image.
+	 * @param int    $height The height of the image.
+	 * @param bool   $recycle Whether to recycle the image file.
+	 */
+	public function with_real_thumbnail( string $file = null, int $width = 640, int $height = 480, bool $recycle = true ): static {
+		return $this->with_middleware(
+			function ( array $args, Closure $next ) use ( $file, $width, $height, $recycle ) {
+				$post = $next( $args );
+
+				update_post_meta(
```

Comment:
it is possible one might want to create a real thumbnail but not set it as the post featured image.

Reply:
Or is this already possible today?

Reply:
Yeah they can just use the attachment factory themselves. I documented that